src/lib/Menu/ContentRightSidebarBuilder.php

```diff
@@ -223,34 +223,32 @@ public function createStructure(array $options): ItemInterface
             ),
         ]);
 
-        if ((new ContentIsUser($this->userService))->isSatisfiedBy($content)) {
+        $contentIsUser = (new ContentIsUser($this->userService))->isSatisfiedBy($content);
+        if ($contentIsUser && $canDelete) {
             $menu->addChild(
                 $this->createMenuItem(
                     self::ITEM__DELETE,
                     [
                         'extras' => ['icon' => 'trash'],
-                        'attributes' => $canDelete
-                            ? $deleteAttributes
-                            : array_merge($deleteAttributes, ['disabled' => 'disabled']),
+                        'attributes' => $deleteAttributes,
```

Comment:
I believe `$deleteAttributes` variable is no longer needed as it's only used in on place - inside above `if` statement.